packages/contract-addresses/addresses.json

```diff
@@ -1,95 +1,103 @@
         "multiAssetProxy": "0xb34cde0ad3a83d04abebc0b66e75196f22216621",
         "staticCallProxy": "0xe1b97e47aa3796276033a5341e884d2ba46b6ac1",
         "erc1155Proxy": "0x19bb6caa3bc34d39e5a23cedfa3e6c7e7f3c931d",
-        "devUtils": "0x326a303dc0fe20f43acb8e5dec99154e47f32d01",
+        "devUtils": "0xfe3b67d48c723988c81fe918da6dd4fb91459d22",
         "zrxVault": "0xa5bf6ac73bc40790fc6ffc9dbbbce76c9176e224",
-        "staking": "0x344d4f661a82afdd84d31456c291822d90d5dc3a",
+        "staking": "0x7cbe3c09cba24f26db24d9100ee915fa9fa21f5b",
         "stakingProxy": "0xc6ad5277ea225ac05e271eb14a7ebb480cd9dd9f",
-        "erc20BridgeProxy": "0x31b8653642110f17bdb1f719901d7e7d49b08141"
+        "erc20BridgeProxy": "0x31b8653642110f17bdb1f719901d7e7d49b08141",
+        "uniswapBridge": "0x0000000000000000000000000000000000000000",
+        "eth2DaiBridge": "0x0000000000000000000000000000000000000000"
     },
     "42": {
         "erc20Proxy": "0xf1ec01d6236d3cd881a0bf0130ea25fe4234003e",
         "erc721Proxy": "0x2a9127c745688a165106c11cd4d647d2220af821",
         "zrxToken": "0x2002d3812f58e35f0ea1ffbf80a75a38c32175fa",
         "etherToken": "0xd0a1e359811322d97991e03f863a0c30c2cf029c",
         "exchangeV2": "0x30589010550762d2f0d06f650d8e8b6ade6dbf4b",
-        "exchange": "0xca8b1626b3b7a0da722ca9f264c4630c7d34d3b8",
+        "exchange": "0x4eacd0af335451709e1e7b570b8ea68edec8bc97",
         "assetProxyOwner": "0x0000000000000000000000000000000000000000",
         "zeroExGovernor": "0x6ff734d96104965c9c1b0108f83abc46e6e501df",
-        "forwarder": "0xd6330f9d2073e1889a295dd1dd2e28d42dec4bff",
+        "forwarder": "0xf6b373c1b783aa435fcdd3a2ce7655f45c311f0a",
         "orderValidator": "0x0000000000000000000000000000000000000000",
         "dutchAuction": "0x0000000000000000000000000000000000000000",
         "coordinatorRegistry": "0x09fb99968c016a3ff537bf58fb3d9fe55a7975d5",
-        "coordinator": "0x10e0b1c2e6065ec7f290c7e3731264f9a2bf2b2d",
+        "coordinator": "0xd29e59e51e8ab5f94121efaeebd935ca4214e257",
         "multiAssetProxy": "0xf6313a772c222f51c28f2304c0703b8cf5428fd8",
         "staticCallProxy": "0x48e94bdb9033640d45ea7c721e25f380f8bffa43",
         "erc1155Proxy": "0x64517fa2b480ba3678a2a3c0cf08ef7fd4fad36f",
-        "devUtils": "0x5078321eec0fd8db5d61d25cce3ea2d6379bb5df",
+        "devUtils": "0x80101f90446d9e19a80109ac2767c8c957debb29",
         "zrxVault": "0xf36eabdfe986b35b62c8fd5a98a7f2aebb79b291",
-        "staking": "0x89150f5eed50b3528f79bfb539f29d727f92821c",
+        "staking": "0x32b06d5611a03737a5f1834a24ccd641033fd89c",
         "stakingProxy": "0xbab9145f1d57cd4bb0c9aa2d1ece0a5b6e734d34",
-        "erc20BridgeProxy": "0xfb2dd2a1366de37f7241c83d47da58fd503e2c64"
+        "erc20BridgeProxy": "0xfb2dd2a1366de37f7241c83d47da58fd503e2c64",
+        "uniswapBridge": "0x0000000000000000000000000000000000000000",
+        "eth2DaiBridge": "0x0000000000000000000000000000000000000000"
     },
     "1337": {
```

Comment:
@abandeali1 @dekz do any addresses need to be updated for `1337`? 

We updated the coordinator server's dependencies today and it caused some strange bugs. We found that downgrading to version `4.4.0-beta.2` of the migrations package resolved the issue.